protos/response.proto

```diff
@@ -24,6 +24,16 @@ message GetDBStatusResponse {
   bool exist = 2;
 }
 
+message GetDBIndexResponseEnvelope {
+  GetDBIndexResponse response = 1;
+  bytes signature = 2;
+}
+
+message GetDBIndexResponse {
+  ResponseHeader header = 1;
+  bytes index = 2;
```

Comment:
Why not a JSON string?
If you decide to keep it binary at least document that this is the result of  `json.Marshal` on the json text...